src/util.h

```diff
@@ -558,8 +559,8 @@ inline QString DirectoryTypeToString ( EDirectoryType eAddrType )
 {
     switch ( eAddrType )
     {
-    case AT_CUSTOM:
-        return QCoreApplication::translate ( "CClientSettingsDlg", "Custom" );
+    case AT_NONE:
+        return QCoreApplication::translate ( "CServerDlg", "Unregistered" );
```

Comment:
```suggestion
        return QCoreApplication::translate ( "CServerDlg", "Private/Unregistered" );
```

Reply:
AT_NONE could, better should say Private/Unregistered since we‘re talking about private servers in the docs – even though it’s not the exact technical description. However I think private is quicker to understand than unregistered.

Will test it as soon as I find time

Reply:
As above.